releases/v20.2.0.md

```diff
@@ -0,0 +1,157 @@
+- The `--socket` flag of [`cockroach start`](../v20.2/cockroach-start.html) was deprecated in v20.1 and has been removed in v20.2. Use `--socket-dir` instead.
+- The textual error and warning messages displayed by [`cockroach quit`](../v20.2/cockroach-quit.html) under various circumstances have been updated. Meanwhile, the message "`ok`" remains as an indicator that the operation has likely succeeded.
+- [`cockroach quit`](../v20.2/cockroach-quit.html) now prints out progress details on its standard error stream, even when `--logtostderr` is not specified. Previously, nothing was printed on standard error. Scripts that wish to ignore this output can redirect the stderr stream.
+- Previously, the phase of server shutdown responsible for range lease transfers to other nodes would give up after 10000 attempts of transferring replica leases away, regardless of the value of `server.shutdown.lease_transfer_wait`. The limit of 10000 attempts has been removed, so that now only the maximum duration `server.shutdown.lease_transfer_wait` applies.
+- Previously, issuing a `SIGTERM` signal twice or after another signal initiated a hard shutdown for a node. Now the first `SIGTERM` signal initiates a graceful shutdown and further occurrences of `SIGTERM` are ignored. To initiate a hard shutdown, issue `SIGINT` two times (or issue a `SIGINT` signal once after a `SIGTERM` signal).
+- Clusters running alphas of 20.2 that use `ENUM` types will not be able to upgrade to betas or major releases of 20.2 due to internal representation changes.
+- Specifying the same option multiple times in the `WITH` clause of the [`BACKUP`](../v20.2/backup.html) or [`RESTORE`](../v20.2/restore.html) statement now results in an error message. Additionally, quoted option names are no longer allowed.
+- The copy of system and `crdb_internal` tables extracted by [`cockroach debug zip`](../v20.2/cockroach-debug-zip.html) is now written using the TSV format (inside the zip file), instead of an ASCII-art table as previously.
+- The [`SHOW RANGE FOR ROW`](../v20.2/show-range-for-row.html) statement now takes a tuple of the row's index columns instead of the full column set of the row.
+- For expression typing involving only operations on constant literals, each constant literal is now assigned a [`type`](../v20.2/data-types.html) before calculation. Previously, a `type` was assigned only to the final result.
+- The file names for heap profile dumps now use the naming scheme `memprof.<date-and-time>.<heapsize>`. Previously, they were named `memprof.<heapsize>.<date-and-time>`.
+- The Docker image is now based on RedHat's UBI instead of Debian.
+
+### Deprecations
```

Comment:
@bdarnell, could use your help on this section. I found the following "deprecations" called out in earlier 20.2 release notes, but I'm worried we're being inconsistent in what we mean by deprecation. In my opinion, when we announce something as deprecated, we are saying that it still works but is no longer officially supported and will be removed (and thus a backward-incompatible change) in a future version.